packages/wdio-types/src/Capabilities.ts

```diff
@@ -77,15 +84,15 @@ declare global {
             /**
              * Defines the current session’s strict file interactability.
              */
-            strictFileInteractability?: boolean,
+            strictFileInteractability?: boolean;
```

Comment:
Let's not use `;` they are not needed (not only here but also in other parts of the file.

Reply:
`,` → `;`?
this was done by the eslint-prettier setup and autoformat, not by me intentionally...

Reply:
can we revert this?

Reply:
yes, we should have a linter for this but that can be done in a different PR

Reply:
done: all `;` removed